package.json

```diff
@@ -167,6 +167,7 @@
     "nodemon": "^2.0.7",
     "peer-id": "^0.14.2",
     "pg": "^8.5.1",
+    "postcss-load-config": "^3.0.0",
```

Comment:
Really needful?

Reply:
right, unimportant for this pull request